### PR TITLE
Adds mod Zombies-World War 2

### PR DESCRIPTION
**Pull request by RogerCooper**
This replaces my earlier AAZ mod.

### CODE REVIEW

**Comment by RoiEXLab** [comment]
@RogerCooper There's a typo in the url

File: triplea_maps.yaml [hunk 1]
```diff
@@ -2273,3 +2273,12 @@
     Edition.</p>
     <br><p>Key features include combining UK Pacific and ANZAC into one nation, revised
     order of play for nations, victory objectives and 8 rounds of play.</p>
+- mapName: Zombies-World War 2
+  mapCategory: EXPERIMENTAL
+  url: https://github.com/triplea-maps/zombies-World_War_2/archive/master.zip
+  version: 1
+  img: https://raw.githubusercontent.com/triplea-maps/zombies-World_War_2/master/preview.png
```
Comment (DanVanAtta) [inline triplea_maps.yaml:2280]: This URL leads to a 404. 
Navigating to the image in the repo, clicking download and copying that link, the URL is: https://github.com/triplea-maps/zombies-World_War_2/raw/master/Preview.png
The difference is the 'P' vs 'p'. A lower case 'p' is preferred, renaming the image would be a preferred fix.

Comment (RogerCooper) [reply triplea_maps.yaml:2280]: Fixed

Comment (DanVanAtta) [reply triplea_maps.yaml:2280]: Re-ran checks, they passed now  :+1:

**Pull request merged**